packages/next/src/server/next.ts

```diff
@@ -64,9 +64,9 @@ export class NextServer {
       res: ServerResponse,
       parsedUrl?: UrlWithParsedQuery
     ) => {
-      return getTracer().trace(NextServerSpan.getRequestHandler, async () => {
+      return getTracer().trace(NextServerSpan.rootRequestHandler, async () => {
         const requestHandler = await this.getServerRequestHandler()
-        return requestHandler(req, res, parsedUrl)
```

Comment:
any reason for that change?

Reply:
Not really. I was setting the status here when it was root SERVER span.
This change won't change behavior since it's `Promise<void>`.